vscode/src/extension.ts

```diff
@@ -953,7 +955,7 @@ function doActivateWithJDK(specifiedJDK: string | null, context: ExtensionContex
             if (p == nbProcess && code != 0 && code) {
                 vscode.window.showWarningMessage(`${SERVER_NAME} exited with ` + code);
             }
-            if (stdErr?.match(/Cannot find java/) || os.type() === "Windows_NT" ) {
+            if (stdErr?.match(/Cannot find java/) || (os.type() === "Windows_NT" && !deactivated) ) {
```

Comment:
Can we move `"Windows_NT"` to constants file, since it is also used in #251 so that it can be utilized there as well?